app/models/employee.rb

```diff
@@ -12,6 +12,15 @@ def employed_to_or_currently_employed
     if currently_employed && employed_to
       errors.add(:base, "Current employees can't have a finish date")
     end
+    if employed_to && employed_from > employed_to
+      errors.add(:base, "Employment start date can't be before finish date ")
+    end
+    if employed_from && employed_from > Time.now
```

Comment:
I actually added and option on the view to allow select date until today, so this validation maybe will not need. I left for double check.